sdk/keyvault/keyvault-keys/README.md

```diff
@@ -207,6 +207,113 @@ const updatedKey = await client.updateKey(keyName, result.version, { enabled: fa
 await client.deleteKey(keyName);
 ```
 
+## Cryptography
+
+Additionally, a set of cryptography functions that are available through the `CryptographyClient`. Similar to the `KeysClient`, `CryptographyClient` will connect to Azure Key Vault with the provided set of credentials. Once connected, `CryptographyClient` can encrypt, decrypt, sign, verify, wrap keys, and unwrap keys.
```

Comment:
What do you think of:

> This library also offers a set of cryptographic utilities available through...

If it doesn't sound better, disregard!

Reply:
I like it, I'll update